lib/APMSpanContext.js

```diff
@@ -33,6 +41,17 @@ class APMSpanContext {
     getParentId() {
         return this._parentId;
     }
```

Comment:
All of these functions (`getId`, `getTraceId`, and `getParentId`) could be changed to ES6 getter syntax. I would also recommend just naming them using the property name, e.g. `get id () return this._spanId;`. Including the verb in `getId` gives the code a decidedly java-ish feel.